xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
<> vs "" is not a coding convention :) it really matters. <> is system, "" is user. 
in oder for "" to search system, a compiler flag must be set right.

ffmpeg's includes are wrong to be "", dates back to when we carried ffmpeg internally where "" was the correct way.